packages/mephisto-task/README.md

```diff
@@ -46,56 +58,62 @@ function MyApp() {
 The `useMephisoTask` React hook exposes the following fields:
 
 ### `taskConfig`
-A config object passed from the backend.
+An arbitrary task-specific config object passed from the backend.
```

Comment:
Useful to note where this comes from, it's the `get_frontend_args` method of the blueprint.